### PR TITLE
CNDE-1938

### PR DESCRIPTION
**Pull request by joshua-mills-cdw**
## Notes

- Add nolock to ODSE references in preprocessing stored procedure

- Generalize dbo.nrt_interview_columns

- Remove unnecessary columns from dbo.nrt_interview_note

- Add columns to nrt_interview for creating F_INTERVIEW_CASE downstream

## JIRA

- **Related story**: [CNDE-1938](https://cdc-nbs.atlassian.net/browse/CNDE-1938)

## Checklist

- [x] PR focuses on a single story.
- [ ] New unit tests added and ensured they pass.
- [ ] Service has been tested in local and it works as expected.
- [ ] Documentation has been updated for this code change (if needed).
- [ ] Code follows the Java Coding Conventions (https://www.oracle.com/java/technologies/javase/codeconventions-programmingpractices.html).

## Types of changes

What types of changes does this PR introduces?

- [ ] Bugfix
- [x] New feature
- [ ] Breaking change

## Testing

- [ ] Does this PR has >90% code coverage?
- [ ] Is the screenshot attached for code coverage?
- [ ] Does the `gradle build` pass in your local? 
- [ ] Is the `gradle build` logs attached?

### CODE REVIEW

File: liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_interview-001.sql [hunk 1]
```diff
@@ -22,7 +22,12 @@ CREATE TABLE dbo.nrt_interview
     ix_interviewee_role varchar(4000)                                   NULL,
     ix_type             varchar(4000)                                   NULL,
     ix_location         varchar(4000)                                   NULL,
+    investigation_uid   bigint                                          NULL,
```
Comment (upasanapattnaik-eq) [inline liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_interview-001.sql:25]: 👍

File: liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_metadata_columns-001.sql [hunk 2]
```diff
@@ -1,8 +1,8 @@
 IF NOT EXISTS (SELECT 1
                FROM sysobjects
-               WHERE name = 'nrt_interview_columns'
+               WHERE name = 'nrt_metadata_columns'
                  and xtype = 'U')
-CREATE TABLE dbo.nrt_interview_columns
+CREATE TABLE dbo.nrt_metadata_columns
 (
```
Comment (upasanapattnaik-eq) [inline liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_metadata_columns-001.sql:6]: I am not able to comment on the columns that haven't been changed. Are NEW_FLAG, LAST_CHG_TIME, LAST_CHG_USER_ID nullable? If so, let's specify that.

File: liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_interview_note-001.sql [hunk 3]
```diff
@@ -6,7 +6,6 @@ CREATE TABLE dbo.nrt_interview_note
 (
     interview_uid    bigint                                          NOT NULL,
     nbs_answer_uid   bigint                                          NULL,
```
Comment (upasanapattnaik-eq) [inline liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_interview_note-001.sql:8]: Is the answer_val baked into another table or not required?

Comment (joshua-mills-cdw) [reply liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_interview_note-001.sql:8]: answer_val was just not required. A handful of the other columns are the elements parsed out of answer_val.

File: liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_metadata_columns-001.sql (hunk 2 above)
Comment (joshua-mills-cdw) [reply liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_metadata_columns-001.sql:6]: Thank you for pointing that out, fixed

**Review by upasanapattnaik-eq — APPROVED** [review]
LGTM!

**Pull request merged**